### ISSUE TITLE
Run analyzers on mozilla-central pushes

### ISSUE THREAD

**Issue by La0** [mozilla/code-review#211]
We need the analyzers to run on new mozilla-central pushes in order to get the issues present in the repository, and compare that with new patches

Refs #210

**Comment by La0** [comment]
Work notes:

- [Autoland Pulse messages](https://firefox-ci-tc.services.mozilla.com/pulse-messages?bindings%5B0%5D%5Bexchange%5D=exchange%2Ftaskcluster-queue%2Fv1%2Ftask-completed&bindings%5B0%5D%5Bpattern%5D=route.index.gecko.v2.autoland.latest.taskgraph.decision) on firefox-ci frontend

**Comment by La0** [comment]
Current plan is:

1. events listen to generic task group completion: `exchange/taskcluster-queue/v1/task-group-resolved` with `#` yields simple payloads with `taskGroupId`.
2. events retrieve the task definition using the `taskGroupId`, if it's an autoland decision task (`GECKO_HEAD_REPOSITORY=https://hg.mozilla.org/integration/autoland`), trigger a code review bot task with `AUTOLAND_GROUP_id=taskGroupId`
3. the code review bot uses existing logic to parse the tasks in the group, and report the revision as "clean" when the tasks have no issues, otherwise report all issues found as currently.
4. the "clean" information is used to detect if issues are new or not.

**Comment by La0** [comment]
Steps 1, 2 & 3 are done and running in production, [we can see autoland revisions on the frontend](https://code-review.moz.tools/#/?repository=autoland)

Only thing left is using that info in the backend/bot decision

**Comment by marco-c** [comment]
We also need https://github.com/mozilla/code-review/issues/211#issuecomment-549899075, e.g. clang-tidy would need to run on autoland too (ignoring failures, so Tier 3).

**Comment by La0** [comment]
Right, because now we basically have 0 issues on autoland detected

**Comment by marco-c** [comment]
Oh, and also we need to support full runs to pre-populate the DB.
